src/services/ValidatorFactoryAJV.js

```diff
@@ -2,7 +2,7 @@
 
 var app = angular.module('app', false);
 
-// Handles draft-04, v5-unofficial
+// Handles draft-04, v5-unofficial, and draft-06
```

Comment:
Perhaps best to say "draft-04 and up" as with your adjustments to the About dialog template?